### PR TITLE
Allow scale-testing PRs using kubemark-5000

### PR DESCRIPTION
**Pull request by shyamjvs**
/cc @wojtek-t @porridge @krzyzacy

### CODE REVIEW

File: prow/config.yaml [hunk 1]
```diff
@@ -2805,6 +2805,79 @@ presubmits:
...
           path: /mnt/disks/ssd0/docker-graph
 
+  - name: pull-kubernetes-kubemark-e2e-gce-scale
+    agent: kubernetes
+    always_run: false
+    max_concurrency: 1
+    skip_branches:
+    - release-1.6  # not supported
+    - release-1.7  # per-release image
+    - release-1.8  # per-release image
```
Comment (krzyzacy) [inline prow/config.yaml]: nit: 1.9 as well? or you want to do `branches: master` instead?

**Comment by krzyzacy** [comment]
lgtm, will let scalability team to apply the label

**Comment by BenTheElder** [comment]
Needs `hack/update-config.sh` (auto security repo versions + auto sort)

File: prow/config.yaml (hunk 1 above)
Comment (shyamjvs) [reply prow/config.yaml]: Done.. thanks!

**Pull request merged**